gson/src/main/java/com/google/gson/FieldNamingPolicy.java

```diff
@@ -20,10 +20,11 @@
 import java.util.Locale;
 
 /**
- * An enumeration that defines a few standard naming conventions for JSON field names.
- * This enumeration should be used in conjunction with {@link com.google.gson.GsonBuilder}
```

Comment:
I imported the eclipse code formatter that was in the project, and had it autoformat on save...  I assumed it was required.  I'll turn it off and revert this.  Thanks